### ISSUE TITLE
Adwaita-dark is not respected

### ISSUE THREAD

**Issue by lionirdeadman** [RobLoach/net.lutris.Lutris#31]
Okay so this is really weird. Adwaita normal with the dark theme ticked works.. but Adwaita-dark with the dark theme ticked or unticked is always the light theme. :thinking:

**Comment by Saroufim** [comment]
Is this an issue only with Lutris? Flatpak doesn't support all themes. The stock adwaita is supported and thus its dark variant would work. Adwaita-dark might be identified as a different theme and it might not work.

**Comment by gasinvein** [comment]
I can't reproduce it, dark variant works for both Adwaita and third-party themes.
Adwaita is part of GTK, so it can't be missing, making this is even more wierd.

**Comment by gasinvein** [comment]
I guess Adwaita-dark as a separate theme is included in the Gnome runtime, but not in the FD.o runtime we switched to in #21

**Comment by LeandroStanger** [comment]
@lionirdeadman Does this bug still happen?

**Comment by lionirdeadman** [comment]
Yep, I just grabbed the latest build made by the Flathub build and it still happens.
